src/main/java/seedu/letsgethired/model/application/InternApplication.java

```diff
@@ -90,7 +91,7 @@ public boolean equals(Object other) {
 
     @Override
     public int hashCode() {
-        // use this method for custom fields hashing instead of implementing your own
+        // No comparing notes because notes is not a strict differentiating factor
         return Objects.hash(company, role, cycle, status);
```

Comment:
Perhaps we can consider creating an explicit test case to show that 2 Intern applications can be equal even if they have different notes!